nimble/controller/src/ble_ll_conn_hci.c

```diff
@@ -850,8 +850,11 @@ ble_ll_conn_hci_update(uint8_t *cmdbuf)
         return BLE_ERR_CMD_DISALLOWED;
     }
 
-    /* See if this feature is supported on both sides */
-    if ((connsm->conn_features & BLE_LL_FEAT_CONN_PARM_REQ) == 0) {
+    /* See if this feature is supported on both sides.
+     * If features has not be exchanged let us give a try and send request
+     */
+    if (connsm->conn_features &&
+        (connsm->conn_features & BLE_LL_FEAT_CONN_PARM_REQ) == 0) {
```

Comment:
I don't understand why this is required - it does not seem right.

Reply:
well this patch is indeed not needed as our controller always does feature exchange. Will remove it.